vivarium/core/process.py

```diff
@@ -53,7 +55,8 @@ def deserialize_value(value):
     elif isinstance(value, str):
         try:
             return serializer_registry.access('units').deserialize(value)
-        except:
+        except Exception:  # TODO(jerry): Narrow this too broad exception
+            log.exception("serializer_registry", exc_info=True, stack_info=True)
```

Comment:
This is suppose to add units with the `units` serializer, only if the units have been saved to the value as a string (which happens upon serializing units).

Reply:
What exception is it raising? Should it catch that exception and not log it, i.e. is it not an error?

The reason for logging is to at least not hide and swallow any arbitrary exception that this could catch. E.g. if you hit ^C it'll raise `InterruptedError` or something.

Reply:
If a value has not been serialized, we'll get `pint.errors.UndefinedUnitError`. This isn't an error. It just means we should use the raw value instead of trying to deserialize it

Reply:
Great! I narrowed the catch to `UndefinedUnitError`.